public/app/plugins/datasource/prometheus/configuration/AlertingSettingsOverhaul.tsx

```diff
@@ -19,9 +21,12 @@ export function AlertingSettingsOverhaul<T extends AlertingConfig>({
   const theme = useTheme2();
   const styles = overhaulStyles(theme);
 
+  const prometheusConfigOverhaulAuth = config.featureToggles.prometheusConfigOverhaulAuth;
+
+  const space = prometheusConfigOverhaulAuth ? styles.alertingTop : '';
+
   return (
-    <>
-      <h3 className="page-heading">Alerting</h3>
+    <ConfigSubSection title="Alerting" className={`${styles.container} ${space}`}>
```

Comment:
there is a utility called `cx`, part of `@emotion/css` that could help do that

non blocking

Reply:
Added!